### PR TITLE
✨support for amp-embed type=yahoofedads

### PR DESCRIPTION
**Pull request by neversay**
Adds support to render Yahoo Native-Display Ad Federation units using the amp-embed tag.

Sample Rendering
Screen Shot 2020-05-15 at 16 34 17

Test page on localhost:
http://localhost:8000/examples/ads.amp.html?type=yahoofedads

### CODE REVIEW

**Comment by neversay** [comment]
@googlebot I signed it!

**Comment by neversay** [comment]
@powerivq and @estherkim , it looks like the saucelab test takes forever to wait in queue, could you help me check what I need to do to make this PR runs the integration test if you have time?

**Comment by estherkim** [comment]
Hey @neversay, we're temporarily allowing Sauce Labs test failures because we've been experience connection issues with Sauce Labs. You can retry the job here, https://travis-ci.org/github/ampproject/amphtml/jobs/688635694

But you should be able to merge this PR regardless.

**Comment by neversay** [comment]
@estherkim thank you, could you help me merge it?

File: ads/yahoofedads.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
```
Comment (estherkim) [inline ads/yahoofedads.js:2]: nit: 2020

File: ads/yahoofedads.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
+<!---
+Copyright 2019 The AMP HTML Authors. All Rights Reserved.
```
Comment (estherkim) [inline ads/yahoofedads.md:2]: nit: 2020

**Comment by neversay** [comment]
@estherkim and @powerivq , it looks like the PR is passing all stages except the saucelab. could you merge this PR if you think it is good to merge? Thank you.

**Pull request merged**